Chat Reporting Helper/assets/fo/lang/ru_ru.json

```
@@ -0,0 +1,30 @@
+{
+    "gui.socialInteractions.tooltip.report": "Пожаловаться на игрока в Mojang",
+    "gui.socialInteractions.narration.report": "Пожаловаться на %s в Mojang",
```

```suggestion
    "gui.socialInteractions.narration.report": "Пожаловаться на игрока %s в Mojang",
```
That's more like how it appears [on the original](https://github.com/InventivetalentDev/minecraft-assets/blob/1.19.3/assets/minecraft/lang/ru_ru.json#L3361) 🤷